docs/topics/images.rst

```diff
@@ -197,3 +197,63 @@ Wagtail comes with three pre-defined image formats, but more can be defined in P
     The CSS classes added to images do **not** come with any accompanying stylesheets, or inline styles. e.g. the ``left`` class will do nothing, by default. The developer is expected to add these classes to their front end CSS files, to define exactly what they want ``left``, ``right`` or ``full-width`` to mean.
 
 For more information about image formats, including creating your own, see :ref:`rich_text_image_formats`
+
+Output image format
+-------------------
+
+Wagtail may automatically change the format of some images when they are resized:
+
+ - PNG and JPEG images always won't change format
```

Comment:
Thanks! I'll also remove always/never since this isn't the case if you've overridden it!